src/core/public/application/types.ts

```diff
@@ -40,15 +40,20 @@ export interface AppBase {
    */
   title: string;
 
+  /**
+   * The status of the application.
+   */
+  status?: AppStatus;
+
   /**
    * An ordinal used to sort nav links relative to one another for display.
    */
   order?: number;
 
   /**
-   * An observable for a tooltip shown when hovering over app link.
+   * A tooltip shown when hovering over app link.
    */
-  tooltip$?: Observable<string>;
+  tooltip?: string;
```

Comment:
Reverted the tooltip observable, as it was currently unimplemented, and should now be updated using the new `AppStatusUpdater` mechanism